nemo/utils/exp_manager.py

```diff
@@ -66,7 +66,8 @@ class CallbackParams:
     mode: Optional[str] = "auto"
     period: Optional[int] = 1
     prefix: Optional[str] = None  # If None, exp_manager will attempt to handle the filepath
-    nemo_save_best_model: bool = False
+    postfix: str = ".nemo"
+    save_best_model: bool = False
 
```

Comment:
Why rename this? Actually in TLT, why are you using `NeMoModelCheckpoint`? Why not just use Lightning's `NeMoModelCheckpoint`